src/plugins/intel_npu/src/plugin/npuw/serialization.cpp

```diff
@@ -261,3 +266,70 @@ void ov::npuw::s11n::read_any(std::istream& stream, ov::Any& var) {
         NPUW_ASSERT(false && "Unsupported type");
     }
 }
+
+void ov::npuw::s11n::read(std::istream& stream, ov::npuw::weights::LazyTensor& var) {
+    var = ov::npuw::weights::LazyTensor::deserialize(stream);
+}
+
+// Weightless
+// FIXME: all serialization needs a good rewriting
```

Comment:
Agree, might be first we need to dissimilate different bools flags with creating of consts and writing them instead of `false` and `true`